docs/examples/payloads/nats-payload.json

```diff
@@ -0,0 +1 @@
+{"metrics":[{"timestamp":"{{TimeStamp}}","metadata":{"id":"{{ID}}"},"data":{"string":"{{Random 5 10}}","float":1.11,"integer":{{Count}}}}]}
```

Comment:
Nitpick: I don't know nats, but this seems to be invalid JSON. If it is not JSON, change the file name of the file to a more appropriate type.

Reply:
Technically correct is the best kind of correct.